book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :author, :title, :rentals
+
+  def initialize(author, title)
+    @author = author
+    @title = title
+    @rentals = []
+  end
```

Comment:
- [ ] A book has many rentals, kindly consider adding this association by creating a method add_rental
e.g
```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```